### PR TITLE
qa/cephfs: don't test kclient on RHEL 7

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
https://tracker.ceph.com/issues/43539

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_7.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-.qa/distros/all/rhel_7.yaml
```
Comment (batrick) [inline qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_7.yaml]: This link needs to be changed to `.qa/distros/all/rhel_8.yaml`.

Comment (rishabh-d-dave) [reply qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_7.yaml]: Fixed.

**Comment by rishabh-d-dave** [comment]
> Also need to copy the directory this link points to
> 
> `qa/cephfs/mount/kclient/overrides/distro/random/supported$`
> 
> into a new directory
> 
> `qa/cephfs/mount/kclient/overrides/distro/random/supported`
> 
> but exclude `rhel_8.yaml`.

Incorporaed the change.

File: qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_8.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+.qa/distros/supported/rhel_latest.yaml
```
Comment (batrick) [inline qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_8.yaml]: `.qa/distros/all/rhel_8.yaml`

**Pull request merged**